src/public/services/__tests__/FieldVerificationService.test.ts

```diff
@@ -0,0 +1,136 @@
+import mockAxios from 'jest-mock-axios'
+
+import {
+  createTransactionForForm,
+  FetchNewTransactionResponse,
+  JsonDate,
+  resetVerifiedField,
+  TRANSACTION_ENDPOINT,
+  triggerSendOtp,
+  verifyOtp,
+} from '../FieldVerificationService'
```

Comment:
this is a matter of personal preference, but I prefer the `import *` syntax for services

Reply:
Will result in verbose usage though (because everything is being exported out from the service file... maybe we should split up the types/constants away? Though I don't think that's the good idea for now since want to encapsulate Typescript code in their own file for ease of migration later)

```ts
      // Arrange
      const expected: FieldVerificationService.FetchNewTransactionResponse = {}
      const mockFormId = 'mockFormId2'

      // Act
      const actualPromise = FieldVerificationService.createTransactionForForm(
        mockFormId,
      )
      mockAxios.mockResponse({ data: expected })
      const actual = await actualPromise

      // Assert
      expect(actual).toEqual(expected)
      expect(mockAxios.post).toHaveBeenCalledWith(
        FieldVerificationService.TRANSACTION_ENDPOINT,
        {
          formId: mockFormId,
        },
      )
```

Reply:
okie for tests it doesn't really matter